lib/lita/handlers/api/lunch.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require 'json'
+
+module Lita
+  module Handlers
+    module Api
+      class Lunch < Lita::Handlers::Api::ApiController
+        def initialize(robot)
+          super
+          @karmanager = Lita::Services::Karmanager.new(redis)
+          @assigner = Lita::Services::LunchAssigner.new(redis, @karmanager)
+        end
+
+        http.get '/winning_lunchers', :winning_lunchers
+        http.get '/current_lunchers', :current_lunchers
+        http.post '/current_lunchers', :opt_in
+
+        def winning_lunchers(request, response)
```

Comment:
Unused method argument - request. If it's necessary, use _ or _request as an argument name to indicate that it won't be used.